src/pages/MerchandiseList/Card.tsx

```diff
@@ -25,7 +25,12 @@ const Card = ({ productId, imgSrc, text, price, sizeRange, isOutOfStock }: CardP
           overflow="hidden"
           _groupHover={{ boxShadow: "xl" }}
         >
-          <Image src={imgSrc} w="100%" maxHeight="300" fallbackSrc="https://via.placeholder.com/150" />
+          <Image 
+            src={imgSrc} 
+            boxSize='300px'
```

Comment:
Setting it to 300px makes the image not centered horizontally on some higher resolution screens.